scapy/contrib/gtp_v2.py

```diff
@@ -633,10 +675,21 @@ class IE_APN(gtp.IE_Base):
                                       length_from=lambda x: x.length)]
 
 
+class IE_BearerTFT(gtp.IE_Base):
+    name = "IE Bearer TFT"
+    fields_desc = [ByteEnumField("ietype", 84, IEType),
+                   FieldLenField("length", None, length_of="Bearer TFT",
+                                 adjust=lambda pkt, x: x + 4, fmt="H"),
+                   BitField("CR_flag", 0, 4),
+                   BitField("instance", 0, 4),
+                   StrLenField("Bearer TFT", "",
+                               length_from=lambda x: x.length)]
+
+
 class IE_AMBR(gtp.IE_Base):
     name = "IE AMBR"
     fields_desc = [ByteEnumField("ietype", 72, IEType),
-                   ShortField("length", 12),
+                   ShortField("length", 8),
```

Comment:
This looks incorrect.

Reply:
@guedou  8 is the correct value. This is a fixed length IE. And the length should represent the length of an element; not an entire IE. BTW, I tested my code against a production level GTPv2 packet and it encodes/decodes fine.